internal_test/mySQL_test/database_test.go

```diff
@@ -1,42 +1,169 @@
 package mySQL_test
 
 import (
+	"database/sql"
+	_ "fmt"
+	"log"
+	"os"
 	"testing"
-
+	_ "github.com/go-sql-driver/mysql"
 	"github.com/matcha-devs/matcha/internal/mySQL"
 )
 
-var db = mySQL.Open()
+// This a variable to hold the test database connection,
+// and run all functions relating to the database:
+var test_db_connection *sql.DB
 
-// TODO(@SeoYoung Cho):what do is the panic error you are having?:
+// Mock environment variable for testing
+// Helper function to create a test database connection and ensure it's clean
+func setupTestDB(t *testing.T) *mySQL.Database {
+	// Get the password from the environment, currently using the admin password this may change in the future!!!
+	password := os.Getenv("MYSQL_PASSWORD")
```

Comment:
Since this will be a constant, no need to call upon the set of each unit test, make a constant for the whole package instead.

Reply:
Yes, this is persistent issue that needs to be addressed in the future. However current implementation needs to remain till refactoring, as dropping a schema table is necessary, for setting up the environment.

Reply:
Agreed, added as a note to the PR.